frontends/mit-open/src/pages/HomePage/TestimonialsSection.tsx

```diff
@@ -0,0 +1,284 @@
+  minWidth: "948px",
+  maxWidth: "948px",
+  height: "326px",
+  backgroundColor: theme.custom.colors.white,
+  color: theme.custom.colors.black,
+  display: "flex",
+  borderRadius: "8px",
+  margin: "0 0 26px 24px",
+  [theme.breakpoints.down("md")]: {
+    minWidth: "311px",
+    maxWidth: "311px",
+    height: "411px",
+    flexDirection: "column",
+    margin: "0",
+  },
+})
+
+const TestimonialCardImage = styled.div({
+  width: "300px",
+  height: "326px",
+  ["img"]: {
+    width: "300px",
+    height: "326px",
+    objectFit: "cover",
+    borderTopLeftRadius: "8px",
+    borderBottomLeftRadius: "8px",
+    [theme.breakpoints.down("md")]: {
+      width: "100%",
+      height: "190px",
+      borderTopRightRadius: "8px",
+      borderBottomLeftRaidus: "0",
+    },
+  },
+  [theme.breakpoints.down("md")]: {
+    width: "100%",
+    height: "190px",
+    borderRadius: "8px 0 8px 0",
+    backgroundColor: theme.custom.colors.darkBlue,
+  },
+})
+
+const TestimonialCardQuote = styled.div({
+  width: "648px",
+  height: "326px",
+  backgroundColor: theme.custom.colors.white,
+  color: theme.custom.colors.black,
+  padding: "0 32px 32px",
+  flexDirection: "column",
+  justifyContent: "space-between",
+  alignItems: "flex-start",
+  flex: "1 0 0",
+  alignSelf: "stretch",
+  borderRadius: "8px",
+  display: "flex",
+  [theme.breakpoints.down("md")]: {
+    width: "100%",
+    height: "161px",
+    borderRadius: "0 8px 0 8px",
+    marginTop: "16px",
+    marginBottom: "16px",
+    padding: "0 16px",
+  },
+
+  ["div.testimonial-quote-opener"]: {
+    color: theme.custom.colors.mitRed,
+    fontStyle: "normal",
+    height: pxToRem(80),
+    width: "100%",
+    fontSize: pxToRem(80),
+    lineHeight: pxToRem(120),
+    [theme.breakpoints.down("md")]: {
+      fontSize: pxToRem(80),
+      height: pxToRem(50),
+      lineHeight: "normal",
+    },
+  },
+
+  ["h4"]: {
+    flexGrow: "1",
+    [theme.breakpoints.down("md")]: {
+      width: "100%",
+      ...theme.typography.subtitle1,
+    },
+  },
+
+  ["div.testimonial-quote-closer"]: {
+    textAlign: "right",
+    width: "100%",
+  },
+})
+
+const TestimonialFadeRight = styled.div({
+  width: "972px",
+  height: "414px",
+  position: "absolute",
+  right: "0",
+  bottom: "0",
+  ["div"]: {
+    marginLeft: "auto",
+    height: "414px",
+    width: "246px",
+    background:
+      "linear-gradient(90deg,rgb(117 0 20 / 0%) 0%,rgb(117 0 20 / 95%) 100%)",
+  },
+  [theme.breakpoints.down("md")]: {
+    display: "none",
+  },
+})
+
+const TestimonialFadeLeft = styled.div({
+  width: "972px",
+  height: "414px",
+  position: "absolute",
+  left: "0",
+  ["div"]: {
+    height: "414px",
+    width: "246px",
+    background:
+      "linear-gradient(270deg,rgb(117 0 20 / 0%) 0%,rgb(117 0 20 / 95%) 100%)",
+  },
+  [theme.breakpoints.down("md")]: {
+    display: "none",
+  },
+})
+
+const TestimonialsSection: React.FC = () => {
+  return (
+    <Section>
+      <Container id="hamster-noises">
```

Comment:
Squeak Squeak.

`<Container> ... </Container>` is https://mui.com/material-ui/react-container/ and ensures the contents have max width at various breakpoints.

However, for this carousel, we DO want fully true width. So the carousel should not be inside `Container`.

Note: Changed on my experimental `slick` branch.